source/community/reactnative/android/app/build.gradle

```diff
@@ -121,17 +159,31 @@ android {
         }
     }
     buildTypes {
+        debug {
+            signingConfig signingConfigs.debug
+        }
         release {
+            // Caution! In production, you need to generate your own keystore file.
+            // see https://facebook.github.io/react-native/docs/signed-apk-android.
+            signingConfig signingConfigs.debug
```

Comment:
This might get missed. Better to provide a proper signingConfig or omit it instead of providing debug config

Reply:
Commented out this line.